jobs/build/refresh-images/Jenkinsfile

```diff
@@ -112,14 +135,76 @@ images:update-dockerfile
   --push
   """
 
-                buildlib.oit """
+//
+// start retry region
+//
+                BUILD_CONTINUED = false
+                waitUntil {
+                    try {
+                        exclude = ""
+                        if (BUILD_EXCLUSIONS != "") {
+                            exclude = "-x ${BUILD_EXCLUSIONS} --ignore-missing-base"
```

Comment:
`${exclude}` needs to be included in the pre-existing oit invocations (e.g. images:build below).

Reply:
Yep  thanks.  Done